annotation-file-utilities/src/annotator/find/AnnotationInsertion.java

```diff
@@ -10,7 +10,12 @@
     /**
      * The annotation to insert.
      */
-    private final String annotation;
+    private final String fullyQualifiedAnnotation;
```

Comment:
Can you update the javadoc for this field and explain the difference between `fullyQualifiedAnnotation` and `annotationFullyQualifiedName`? The field names sound very similar.

Reply:
Done.